### PR TITLE
Single video page and notes functionality

### PR DESCRIPTION
**Pull request by stutisk**
added single video page functionality

### CODE REVIEW

**Review by abhij1607 — APPROVED** [review]
Neat and clean code
Good folder structure
Can be merged

File: src/Pages/SingleVideoPage/SingleVideoPage.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    
+    setModal(true) && <PlaylistModal />;
+  
+};
+
+  useEffect(() => {
+    async function singleVideo() {
+      try {
+        const res = await axios.get(`/api/video/${videoId}`);
+        console.log(res.data);
```
Comment (prachi-sahani) [inline src/Pages/SingleVideoPage/SingleVideoPage.jsx:27]: can be removed

**Review by prachi-sahani — COMMENTED** [review]
Overall the code looks clean
Good to merge

File: src/Pages/SingleVideoPage/SingleVideoPage.jsx (hunk 1 above)
Comment (stutisk) [reply src/Pages/SingleVideoPage/SingleVideoPage.jsx:27]: yes will remove it

**Pull request merged**